### PR TITLE
fix(#75): commented code and minor refactoring changes

### PR DESCRIPTION
**Pull request by sigbbe**

### CODE REVIEW

File: src/components/common/Carousel/Carousel.tsx [hunk 1]
```diff
@@ -67,6 +96,27 @@ const Carousel: FC<CarouselPropsI> = ({
         };
     }, [withAutomaticSliding, page, paginate]);
 
+    // parameters for the animation transition
+    const transition = {
+        x: { type: 'spring', stiffness: 300, damping: 30 },
+        opacity: { duration: 0.2 },
+    };
+
+    // show last page if the horizontal swipe power was negatibe
```
Comment (FredSkau) [inline src/components/common/Carousel/Carousel.tsx:105]: negative*

**Comment by Jorrre** [comment]
Closes #75

File: src/components/home/Solarpanel/Weather.tsx [hunk 2]
```diff
@@ -1,14 +1,24 @@
 import { Box } from '@chakra-ui/react';
 import { motion } from 'framer-motion';
-import { useEffect, useRef } from 'react';
+import { FC, useEffect, useRef } from 'react';
 import cloudIcon from '../../../assets/images/cloudIcon.svg';
 import sunIcon from '../../../assets/images/sunIcon.svg';
 
+/**
+ * boolean states for either showing or not showing the sun and the coud
```
Comment (FredSkau) [inline src/components/home/Solarpanel/Weather.tsx:8]: cloud*

**Review by FredSkau — APPROVED** [review]
Et par skrivefeil, men ellers nais image

**Pull request merged**